mava/systems/jax/ippo/networks.py

```diff
@@ -44,6 +44,7 @@ def __init__(
         self,
         policy_network: networks_lib.FeedForwardNetwork,
         policy_params: networks_lib.Params,
+        policy_init_state: Any,
```

Comment:
Why can't you just make this an optional variable defaulting to `None`?

Reply:
Because a user is not not directly interacting with PPONetworks. So I made it not default to anything as a value should always be provided. If it is not provided then it most likely means that there is a bug in the code. Or what do you think?

Reply:
Created a new comment below